app/+QSP/@CohortGeneration/CohortGeneration.m

```diff
@@ -509,7 +598,9 @@ function clearData(obj)
                 obj.SimResults = {};
                 obj.SimFlag = [];
                 
+                obj.Log(['running virtual cohort generation ' obj.Name])
```

Comment:
probably need to remove this since we are using a new logger

Reply:
Yes, but that should be a project independently done from the refactor. It is best to try to keep changes atomic for various reasons, one of which is enabling the possibility of reverting or editing a particular change(s).